### PR TITLE
Update create-cluster-kubeadm.md

### PR DESCRIPTION
**Pull request by naarani**
calico version update

### CODE REVIEW

File: content/en/docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -268,7 +268,7 @@ For Calico to work correctly, you need to pass `--pod-network-cidr=192.168.0.0/1
 
 ```shell
 kubectl apply -f https://docs.projectcalico.org/v3.3/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml
```
Comment (neolit123) [inline content/en/docs/setup/independent/create-cluster-kubeadm.md]: to me it seems that this line is no longer needed.
the 3.4 yaml includes ClusterRoleBinding and ClusterRole objects.

**Comment by Rajakavitha1** [comment]
Thanks for the PR @naarani . Could you please address the suggestions in the review comments.

File: content/en/docs/setup/independent/create-cluster-kubeadm.md (hunk 1 above)
Comment (zparnold) [inline content/en/docs/setup/independent/create-cluster-kubeadm.md]:
```suggestion
```

**Comment by Rajakavitha1** [comment]
@naarani could you please address the changes requested.

**Comment by Bradamant3** [comment]
/close

@naarani if you want to re-open this PR, please feel free, but make sure to address the comments first. Thanks!

**Pull request closed without merging**